airflow-core/src/airflow/ui/src/pages/TaskInstance/Logs/Logs.tsx

```diff
@@ -83,6 +84,9 @@ export const Logs = () => {
     tryNumber: tryNumber === 0 ? 1 : tryNumber,
   });
 
+  const externalLogName = useConfig("external_log_name") as string;
+  const showExternalLogRedirect = useConfig("show_external_log_redirect") as boolean;
```

Comment:
In other places of the code we seem to prefer this construct.
```suggestion
  const showExternalLogRedirect = Boolean(useConfig("show_external_log_redirect"));
```

Reply:
Thanks for pointing out, I would keep in mind~